src/main/resources/application.properties

```
@@ -0,0 +1,10 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/diplom
+spring.datasource.username=postgres
+spring.datasource.password=1234
+spring.jpa.hibernate.ddl-auto=update
```

```suggestion
spring.jpa.hibernate.ddl-auto=validate
```
при использовании liquibase стратегия должна быть validate, иначе смысла в liquibase нет никакого